custom_components/venta/venta_strategy.py

```diff
@@ -119,30 +120,23 @@ def _build_message(
         body = dumps(
             {
                 "Header": {
-                    "DeviceType": self._header.device_type,
```

Comment:
Does it hurt to keep? I would prefer to have it since it's confirmed to work for other devices.